lib/bridge/matrix-handler.js

```diff
@@ -303,9 +319,9 @@ var _onInvite = Promise.coroutine(function*(req, event, inviter, invitee) {
     // mark this room as being processed in case we simultaneously get
     // messages for this room (which would fail if we haven't done the
     // invite yet!)
-    processingInvitesForRooms[event.room_id + event.state_key] = req.getPromise();
-    req.getPromise().finally(function() {
-        processingInvitesForRooms[event.room_id + event.state_key] = undefined;
+    this._processingInvitesForRooms[event.room_id + event.state_key] = req.getPromise();
+    req.getPromise().finally(() => {
+        this._processingInvitesForRooms[event.room_id + event.state_key] = undefined;
```

Comment:
They're for all intents and purposes equivalent. `delete` probably makes the intent clearer though, so will change.